.github/workflows/main.yml

```
@@ -215,14 +136,14 @@ jobs:
           cache-from: type=gha,scope=wikibase-shared
           cache-to: type=gha,mode=max,scope=wikibase-shared
 
-  build-apache-assets-production:
+  build-apache-assets::
```

actionlint (1.7.12)

[error] invalid job ID "build-apache-assets:". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)